docs/authorization.rst

```diff
@@ -58,10 +58,44 @@ class and the safe option.
+
++---------------------------------+------------------+-----------------------------------------------+----------------------+
+| HTTP + URI                      | Method           | User’s permissions required to grant access   | Includes check for   |
++=================================+==================+===============================================+======================+
+| ``POST <resource>/``            | create\_list     | ``add``                                       |                      |
++---------------------------------+------------------+-----------------------------------------------+----------------------+
+| ``POST <resource>/<id>`` (\*)   | create\_detail   | ``add``                                       |                      |
++---------------------------------+------------------+-----------------------------------------------+----------------------+
+| ``GET <resource>/``             | read\_list       | ``change``                                    |                      |
++---------------------------------+------------------+-----------------------------------------------+----------------------+
+| ``GET <resouce>/<id>``          | read\_detail     | ``change``                                    |                      |
++---------------------------------+------------------+-----------------------------------------------+----------------------+
+| ``PUT <resource>/``             | update\_list     | ``change``                                    | ``read_list``        |
++---------------------------------+------------------+-----------------------------------------------+----------------------+
+| ``PUT <resource>/<id>``         | update\_detail   | ``change``                                    | ``read_detail``      |
++---------------------------------+------------------+-----------------------------------------------+----------------------+
+| ``DELETE <resource>/``          | delete\_list     | ``delete``                                    | ``read_list``        |
++---------------------------------+------------------+-----------------------------------------------+----------------------+
+| ``DELETE <resource>/``          | delete\_detail   | ``delete``                                    | ``read_detail``      |
++---------------------------------+------------------+-----------------------------------------------+----------------------+
+
+(*) The permission check for ``create_detail`` is implemented in ``DjangoAuthorization``, however ModelResource does not provide an implementation and raises HttpNotImplemented.
+
+
+Notes:
+
+* The actual permission checked is `<app_label>.<permission>_<model>` where app_label is derived from the resource's model (e.g. `myapp.change_foomodel`)
+* `PUT` may revert to `POST` behavior if the object(s) are not found. In this case the respective `POST` authorizations are required. 
```

Comment:
When PUT tries to create a new model, tastypie treats it as a creation and checks for an "add" permission via `authorized_create_detail` in the `save()` method instead of a "change" permission.